src/layer/S5PL2Layer.ts

```diff
@@ -0,0 +1,63 @@
+import { BBox } from 'src/bbox';
+import { PaginatedTiles } from 'src/layer/const';
+import { DATASET_S5PL2 } from 'src/layer/dataset';
+import { AbstractSentinelHubV3Layer } from 'src/layer/AbstractSentinelHubV3Layer';
+
+/*
+  S-5P is a bit special in that we need to supply productType when searching
+  for tiles, but we don't need it for getMap() (it is determined automatically
+  from the evalscript).
+*/
+
+export enum ProductType {
+  AER_AI = 'AER_AI',
+  CLOUD = 'CLOUD',
+  CO = 'CO',
+  HCHO = 'HCHO',
+  NO2 = 'NO2',
+  O3 = 'O3',
+  SO2 = 'SO2',
+  CH4 = 'CH4',
+}
+
+type S5PL2FindTilesDatasetParameters = {
+  type: string;
+  productType: ProductType;
+};
+
+export class S5PL2Layer extends AbstractSentinelHubV3Layer {
+  public readonly dataset = DATASET_S5PL2;
```

Comment:
Should `minQa` and `timeliness` also be supported? They are used only by processApi and afaik can't be defined in dasboard for ogc request.

Reply:
I thought about that, but since we won't use them yet anyway, I would suggest leaving them for some future MR, if you agree?

Reply:
OK, just create card for this otherwise we'll forget about it as it is not being used yet.